Additions to Curriculum

jadecodes8191 opened the pull request.

jadecodes8191 committed:
Update untitled.md

jadecodes8191 committed:
Update untitled.md

jadecodes8191 committed:
Update untitled.md

jadecodes8191 committed:
Update using-git-effectively.md

jadecodes8191 committed:
Update using-git-effectively.md

jadecodes8191 committed:
Update servos.md

jadecodes8191 committed:
Update servos.md

broad-well commented:
By the way, you can create your own branch on this very repo now---no fork necessary.

Commit:
Update The Autonomous Period for accuracy

If you check out Skystone Game Manual Part 1, you will discover that they changed the name of the Team Marker to the _Team Scoring Element_, perhaps to disambiguate from the new _Alliance Marker_.

Commit:
Update and finish Servos

* The servo page is meant to include information about servos only. For retrieving instances from the `hardwareMap`, I probably need to add that to the "Configuring Hardware" section.
* Also, observe how we use code blocks in Markdown. Many code blocks are in the content of this commit.
* This curriculum assumes in-depth knowledge of Java (see Introduction). This aligns with our philosophy of dedication and specialization. The audience would know to import the interface before using it.

Commit:
Edit and finish TeleOp

Commit:
Fix link and description for Using Git Effectively
